Userland/Libraries/LibWeb/MimeSniff/MimeType.cpp

```diff
@@ -22,11 +22,11 @@ namespace Web::MimeSniff {
 bool is_javascript_mime_type_essence_match(StringView string)
 {
     // A string is a JavaScript MIME type essence match if it is an ASCII case-insensitive match for one of the JavaScript MIME type essence strings.
-    // NOTE: The mime type parser automatically lowercases the essence.
-    auto type = MimeType::parse(string);
-    if (!type.has_value())
-        return false;
-    return type->is_javascript();
+    for (const auto& javascript_essence : s_javascript_mime_type_essence_strings) {
```

Comment:
[East const](https://github.com/LadybirdBrowser/ladybird/blob/master/Documentation/CodingStyle.md#const-placement) should be used for this variable declaration.

Reply:
Right, I am still getting the whole development setup right so I catch those before sending the PR!